vmdk_plugin/main.go

```diff
@@ -97,17 +99,31 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	logLevel := flag.String("log_level", "info", "Logging Level")
 	configFile := flag.String("config", config.DefaultConfigPath, "Configuration file path")
 
+	// Load the configuration if one was provided.
+	c, err := config.Load(*configFile)
+	if err != nil {
+		log.Warning("Failed to load config file %s: %v", *configFile, err)
+	}
+
+	// Default driver, overridden by the config file
+	name := vsphereDriver
+	if c.Driver != "" {
+		name = c.Driver
+	}
+
+	// Driver specified on the command line overrides option in the config file
+	driverName := flag.String("driver", name, "Volume driver")
+
 	// photon driver options
 	targetURL := flag.String("target", "", "Photon controller URL")
```

Comment:
Not related to this change but if user starts Plugin with "photon" driver then we should fail to start if required parameters are missing in CLI or config file.